src/shared/components/Button/Button.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+
+interface Button {
+  className: string;
+  onClick: React.MouseEventHandler<HTMLButtonElement>;
+  buttonText: string;
+}
+
+const Button: React.FC<Button> = ({
```

Comment:
So are you using const or function naming convention? Stick with chosen one.

Reply:
ok Fixed